redmine/manifests/redmine-svc.yaml

```diff
@@ -9,8 +9,9 @@ metadata:
 spec:
   type: LoadBalancer
   ports:
-  - port: 80
-    targetPort: redmine-http
+  - name: http
```

Comment:
Let's stick with http for this, though I agree we can use IANA defined names for other services.